optimum/intel/generation/modeling.py

```diff
@@ -273,13 +273,17 @@ def forward(
                 num_attention_heads = self.normalized_config.num_attention_heads
                 hidden_size = self.normalized_config.hidden_size
                 d_k = hidden_size // num_attention_heads
-
-                if self.config.model_type != "bloom":
+                if self.config.model_type == "gpt_bigcode":

```

Comment:
Could you add a test to verify inference is behaving as expected ? (using `INCModelForCausalLM` can be enough for now)